include/galsim/SBMoffatImpl.h

```diff
@@ -45,26 +45,26 @@ namespace galsim {
         double maxK() const;
         double stepK() const;
 
-        void getXRange(double& xmin, double& xmax, std::vector<double>& ) const 
+        void getXRange(double& xmin, double& xmax, std::vector<double>& ) const
```

Comment:
Yeah, sorry about that.  I have a line in my .emacs that automatically removes all trailing whitespace upon a save.  My own personal OCD I guess.

I see that there are some emacs packages online though that will apparently only do this for lines that I've actually edited.  I'll look into this.

In the meantime..., supposedly adding a `?w=1` to a github url will ignore whitespace changes.  Also I think this is an option for most diff utils.  `-w` for gnu diff.